Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -540,5 +540,26 @@
     <UnlocalizedDescription>furn_rt_generator_desc</UnlocalizedDescription>
   </Furniture>
 
+  <Furniture objectType ="trading_console">
+    <Name>Trading Console</Name>
+    <Description>This trading console allows better trading</Description>
+    <MovementCost>2</MovementCost>
+    <PathfindingModifier>0</PathfindingModifier>
+    <PathfindingWeight>2</PathfindingWeight>
+    <Width>1</Width>
+    <Height>1</Height>
+
+    <PowerConnection inputRate ="1" outputRate ="" capacity =""/>
+    
+    <BuildingJob jobTime ="0"/>
+    
+    <ContextMenuAction FunctionName="TradingConsole_PassivelyTrade" Text="Passively trade" RequiereCharacterSelected="false" />
```

Comment:
Are you looking to implement these functions? if you do you can add them to Furniture.lua